Makefile

```diff
@@ -1,8 +1,32 @@
 SNAPPY = $(CHAIN)/vendor/github.com/google/snappy
 ROCKSDB = $(CHAIN)/vendor/github.com/facebook/rocksdb
+DB_DEV = core
+RAFT_DEV = $(HOME)/.chaincore/raft
 
-null:
-	@echo Please specify a make target.
+default: run
+
+## run a development version of Chain Core at http://localhost:1999
+run:
+	cored
+
+## reset the development database
+resetdb:
```

Comment:
Not sure if you're suggesting that it should or shouldn't, but to clarify, the third line in this task drops the Raft file storage. The intent of this task to is to delete anything that constitutes Chain Core's "database" so you get a clean slate the next time you run Chain Core.